api/src/bankerx/api/PublicEndpoints.scala

```diff
@@ -13,7 +13,11 @@ import bankerx.API.*
 import bankerx.api.fdc3.*
 import bankerx.api.Codecs.{given, *}
 
+/**
+ * Defines how the path will look for the specific endpoints
+ */
 object PublicEndpoints:
+// You can access this endpoint with "www.[host]/api/bank/[bankId]/terms"
```

Comment:
`www` is not necessarily true and isn't in our usecase. I'd say something like:
`(http|https)://[host][path]/api/bank/[bankId]/terms`